File: mobile/src/main/java/ro/antiprotv/sugar/AboutActivity.java
/**
 * Copyright Cristian "ciuc" Starasciuc 2016
 * cristi.ciuc@gmail.com
 */
package ro.antiprotv.redalert;

import android.content.Intent;
import android.graphics.drawable.ColorDrawable;
import android.os.Bundle;
import android.support.v7.app.AppCompatActivity;
import android.text.Html;
import android.view.View;
import android.widget.Button;
import android.widget.TextView;

/**
 * Created by ciuc on 7/19/16.
 */
public class AboutActivity extends AppCompatActivity {
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_about);
        TextView foo = (TextView) findViewById(R.id.aboutText);
        foo.setText(Html.fromHtml(getString(R.string.about_text)));
        getSupportActionBar().setBackgroundDrawable(new ColorDrawable(getResources().getColor(R.color.red)));
        Button back = findViewById(R.id.about_back);
        back.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                Intent list = new Intent();
                list.setClassName(AboutActivity.this, "ro.antiprotv.redalert.AlertListActivity");
                startActivity(list);
            }
        });
    }

    @Override
    protected void onStart() {
        super.onStart();
    }
}

File: mobile/src/main/java/ro/antiprotv/sugar/ui/AboutActivity.kt
/**
 * Copyright Cristian "ciuc" Starasciuc 2016
 * cristi.ciuc@gmail.com
 */
package ro.antiprotv.sugar.ui

import android.content.Intent
import android.graphics.drawable.ColorDrawable
import android.os.Bundle
import android.text.Html
import androidx.appcompat.app.AppCompatActivity
import kotlinx.android.synthetic.main.activity_about.*
import ro.antiprotv.sugar.R

/**
 * Created by ciuc on 7/19/16.
 */
class AboutActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_about)

        aboutText.text = Html.fromHtml(getString(R.string.about_text), Html.FROM_HTML_MODE_COMPACT)
        supportActionBar?.setBackgroundDrawable(ColorDrawable(resources.getColor(R.color.red, theme)))

        button_about_back.setOnClickListener {
            val list = Intent()
            list.setClassName(this@AboutActivity, "ro.antiprotv.redalert.AlertListActivity")
            startActivity(list)
        }
    }
}